### PR TITLE
4 about section

### PR DESCRIPTION
**Pull request by shlok-2003**
This push request resolves issue: #4 
Added:

About TCET
Counter
Events and Awards Swiper

### CODE REVIEW

**Comment by shlok-2003** [comment]
The events and awards swiper are hidden for the mobile deployment, as per the mobile UI in figma

File: src/components/HomePage/About/index.astro [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Awards from "./Awards.jsx"
+---
+
+<section id="about" class="grid grid-cols-1 sm:grid-cols-2 gap-16 justify-center border-t-2 pt-5 lg:pt-0">
+    <div class="px-5 md:pl-10 flex flex-col justify-center items-start">
+        <h1 class="text-gray-900 text-3xl sm:text-4xl font-semibold font-title text-black leading-tight tracking-tight">About TCET</h1>    
+        <h3 class="text-xl sm:text-2xl leading-normal text-gray-600">Arcu dui turpis fringilla bibendum in ullamcorper suscipit sapien volutpat. Vestibulum cursus neque est est facilisis.</h3>
+        <p class="hidden md:block text-gray-500 text-justify text-sm lg:text-base">Arcu dui turpis fringilla bibendum in ullamcorper suscipit sapien volutpat. Vestibulum cursus neque est est facilisis. Nisl eget ut tellus varius lacus. Arcu dui turpis fringilla bibendum in ullamcorper suscipit sapien volutpat. Vestibulum cursus neque est est facilisis. Nisl eget ut tellus varius lacus. Arcu dui turpis fringilla bibendum in ullamcorper suscipit sapien volutpat.</p>
+
+        <a href="/" class="mt-6 py-1.5 px-3.5 border-2 rounded-lg border-gray-400 text-gray-900 font-medium">
```
Comment (JayeshVP24) [inline src/components/HomePage/About/index.astro:13]: change `border-2` to `border`

**Comment by JayeshVP24** [comment]
Delete package-lock.json

**Comment by shlok-2003** [comment]
Refactored the code. You can check now

**Review by JayeshVP24 — APPROVED** [review]
LGTM

**Pull request merged**